src/views/product-downloads-view/helpers.ts

```diff
@@ -142,31 +142,6 @@ export function generateEnterprisePackageManagers(
 	]
 }
 
-export const generatePackageManagers = ({
```

Comment:
Edit: moved this to a separate file. Based on the bundle analysis, which showed a `200 kB+` increase for all `/downloads` routes, it seemed like `@hashicorp/platform-code-highlighting` was getting bundled if I kept it in this helper file (which is used on the client).

Moved this to `server-helpers`, and updated the implementation there to add highlighting, to see if that prevents the bundle size issue.